src/components/UI/Text/Textarea.vue

```diff
@@ -36,26 +36,30 @@
       >
         <inline-svg :src="startIconObj"></inline-svg>
       </div>
-
       <!-- input text area -->
-      <textarea
-        class="p-2 border placeholder-blueGray-300 text-blueGray-600 bg-white disabled:bg-gray-200 rounded text-md border-blueGray-300 focus:outline-none focus:ring focus:border-transparent focus:shadow-outline w-full border-gray-200 disabled:cursor-not-allowed"
+
+      <div
+        contenteditable="true"
+        class="pt-7 h-30 textbox pl-0 border placeholder-blueGray-300 text-blueGray-600 bg-white disabled:bg-gray-200 rounded text-md border-blueGray-300 focus:outline-none focus:ring focus:border-transparent focus:shadow-outline w-full border-gray-200 disabled:cursor-not-allowed"
         :class="[inputAreaClass, boxStyling]"
         :disabled="isDisabled"
+        ref="editor"
```

Comment:
Is `textarea` a better name?
or maybe `inputArea` or `inputTextArea`

I think `editor` is not the most accurate

Reply:
inputArea sounds good. will change this.